test/browser/step_definitions/details.js

```diff
@@ -23,14 +23,6 @@ Then("they should be redirected to the landingPage", async function () {
 });
 
 
-Then("they should be redirected as a success", function () {
```

Comment:
I can see that this Then is used in some of the feature files but its flagged as being removed. Is this a duplicate?

Reply:
thats right its a duplicate step